umap/views.py

```diff
@@ -182,6 +182,30 @@ def get_template_names(self):
 user_maps = UserMaps.as_view()
 
 
+class UserStars(UserMaps):
+    template_name = "auth/user_stars.html"
+
+    def get_context_data(self, **kwargs):
+        owner = self.request.user == self.object
+        manager = Map.objects if owner else Map.public
+        stars = Star.objects.filter(by=self.object).values("map")
+        maps = manager.filter(pk__in=stars)
+        if owner:
+            per_page = settings.UMAP_MAPS_PER_PAGE_OWNER
+            limit = 100
+        else:
+            per_page = settings.UMAP_MAPS_PER_PAGE
+            limit = 50
+        maps = maps.order_by('-modified_at')[:limit]
```

Comment:
Is the `limit` pertinent if you paginate the view anyway?

Reply:
I'd so, otherwise we'd paginate indefinitely, but I may be missing something.